src/MongoDB.Analyzer/Core/Poco/PocoDataFiller.cs

```diff
@@ -30,12 +45,16 @@ private static object GetPropertyOrFieldValue(Type memberType, string memberName
         }
         else if (memberType.IsString())
         {
-            return $"{memberName}_val";
+            return HandleString(memberName);
```

Comment:
Need to fill the numeric values as well.

Reply:
yea I can do the string searching stuff(i.e. using the property/field name) and if I can't find anything i'll just default to the modulus stuff we have now